testing/BUILD.bazel

```diff
@@ -27,6 +27,7 @@ kt_android_library(
         artifact("com.google.truth:truth"),
         artifact("androidx.test.ext:junit"),
         artifact("org.mockito:mockito-core"),
+        artifact('com.google.android.material:material:1.2.0-alpha02'),
         artifact("junit:junit:4.12"),
```

Comment:
Add this according to its lexicographical order.

Reply:
Please remove these artifacts as they are duplicate here. 
If you want to add something new, please follow the third party format.